Makefile

```diff
@@ -21,9 +22,13 @@ lint:
 mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
-tests:
-	PYTHONWARNINGS=ignore:ResourceWarning coverage run --source=terra_notebook_utils \
-		-m unittest discover --start-directory tests --top-level-directory . --verbose
+test: $(tests)
+	coverage combine
+	rm -f .coverage.*
+
+# A pattern rule that runs a single test script
+$(tests): %.py : mypy lint
```

Comment:
Seems like mypy and flake8 are going to be run for the entire codebase for each test when `make test` is run?  Could this be refactored?

Reply:
They do not run for each test.

You can verify by looking for flake8/mypy output from `make  test`